SAMLPortal/Models/GlobalSettings.cs

```diff
@@ -100,12 +109,31 @@
 			context.SaveChanges();
 		}
 
-		public static void UpdateFromDatabase()
+		private static void UpdateFromEnvironment()
 		{
+			_appSettings.Clear();
+
+			// Debug Only:
+			_appSettings.Add("CONFIG_PATH", "/tmp/SAMLPortal");
+
+			IDictionary environment = Environment.GetEnvironmentVariables();
+			foreach (var key in environment.Keys)
+			{
+				if (key.ToString().StartsWith("SP_") && !key.ToString().StartsWith("SP_MYSQL"))
+				{
+					string shortenedKey = key.ToString().Remove(0, 3);
+					_appSettings.Add(shortenedKey, environment[key].ToString());
+				}
+			}
+		}
+
+		private static void UpdateFromDatabase()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'UpdateFromDatabase'.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)